Makefile

```diff
@@ -12,7 +12,7 @@ check: cmd/osde2e-docs
 	go run $(PKG)/$< --check
 	CGO_ENABLED=0 go test -v ./cmd/... ./pkg/... 
 	go get -u github.com/golangci/golangci-lint/cmd/golangci-lint && \
-    golangci-lint run ./... 
+    golangci-lint run -c .golang-ci.yml ./... 
```

Comment:
Actually use the config (I believe it was not auto-detecting the config previously)